paper/paper.md

```diff
@@ -1,124 +1,237 @@
-method operates on a ``datetime`` to get the corresponding solar
-position. The methods combine these data sources when calling the
-function layer, then return the results to the user. The final level of
-API is the ``ModelChain`` class, designed to simplify and standardize
-the process of stitching together the many modeling steps necessary to
-convert a time series of weather data to AC solar power generation,
-given a PV system and a location.
-
-pvlib python was ported from the PVLib MATLAB toolbox in 2014
-[@Stein2012, @Andrews2014]. Efforts to make the project more pythonic
-were undertaken in 2015 [@Holmgren2015]. Additional features continue to
-be added, see, for example [@Stein2016, @Holmgren2016] and the
-documentation's "What's New" section.
-
-pvlib python has been used in numerous studies, for example, of solar
-power forecasting [@Gagne2017, @Holmgren2017], development of solar
-irradiance models [@Polo2016], and estimation of photovoltaic energy
-potential [@Louwen2017]. Mikofski et. al. used pvlib python to study
-the accuracy of clear sky models with different aerosol optical depth
-and precipitable water data sources [@Mikofski2017] and to determine the
-effects of spectral mismatch on different PV devices [@Mikofski2016].
-pvlib python is a foundational piece of an award, "An Open Source
-Evaluation Framework for Solar Forecasting," made under the Department
-of Energy Solar Forecasting 2 program [@DOESF2].
-
-Plans for pvlib python development includes the implementation of new
-and existing models, addition of functionality to assist with
-input/output, and improvements to API consistency.
-
-The source code for each pvlib python version is archived with Zenodo
-[@pvlibZenodo].
 
-# Acknowledgements
-
-The authors acknowledge and thank the code, documentation, and
-discussion contributors to the project.
+# Summary
 
-WH acknowledges support from the Department of Energy's Energy
-Efficiency and Renewable Energy Postdoctoral Fellowship Program
-(2014-2016), Tucson Electric Power, Arizona Public Service, and Public
-Service Company of New Mexico (2016-2018), and University of Arizona
-Institute for Energy Solutions (2017-2018).
+pvlib python is a community-developed, open-source software toolbox
+for simulating the performance of solar photovoltaic (PV) energy
+systems.  It provides reference implementations of
+over 100 empirical and physics-based models from the peer-reviewed scientific literature,
+including solar position algorithms, irradiance models, thermal models,
+and PV electrical models.  In addition to these individual low-level
+model implementations, pvlib python provides high-level workflows
+that chain these models together like building blocks to form complete
+"weather-to-power" photovoltaic system models.  It also provides functions to
+fetch and import a wide variety of weather datasets useful for PV modeling. 
+
+pvlib python has been developed since 2015 and follows modern best
```

Comment:
Is there a reason we're saying 2015 here? The first commit was in 2013: 059ec272601b40affcc3a9c689018eb423ae3380. The first non-trivial commit was in 2014: 454467744fc4553115db05c1afadf1f0f08a5b10.

Reply:
Not sure!  Maybe I meant the first release was in 2015?  Regardless, the text is updated to say "developed since 2013".